run_project_tests.py

```diff
@@ -356,6 +356,13 @@ def have_d_compiler():
         return True
     return False
 
+def have_objc_compiler():
+    if shutil.which("objc") and shutil.which("objcpp"):
+        return True
```

Comment:
The best way is to do what we do in the unit tests: [`Environment.detect_objc_compiler()`](https://github.com/mesonbuild/meson/blob/8b73d807924dde250f1c4340885ef46a22b82ba7/run_unittests.py#L555).